elide-async/src/main/java/com/yahoo/elide/async/service/thread/TableExportThread.java

```diff
@@ -36,33 +41,68 @@
  */
 @Slf4j
 public class TableExportThread implements Callable<AsyncAPIResult> {
+    private static final String FORWARD_SLASH = "/";
```

Comment:
Why do we have both TableExportThread and AsyncQuerythread?  Do we need both concepts?